src/oss/confluent.sh

```diff
@@ -95,6 +95,14 @@ declare -a commands=(
     "config"
 )
 
+declare -a dynamic_cli_commands=(
+    "sr-acl"
```

Comment:
Right. The second way is more flexible. Because if you define now the `acl` command for schema-registry as `sr-acl` when you'll need - let's say hypothetically - acl for Connect, then you have a problem. You'll either have to call it `connect-acl` and have two top level commands that have the namespace hardcoded in their name, which is not acceptable, or deprecate `sr-acl` and add the level of indirection then. 

In any case, we have discussed the namespacing issues recently, and currently we go with the option of defining generics command names (e.g. `load`, `list`, `status`) and then we specialize based on the "context". A good example is `status`: 
`confluent status` with a service name returns the status of a service, `confluent status` with a connector name returns the name of the connector. 

Here `acl` will be like `load`. It will imply Schema-Registry. We've postponed the option of adding a level of specialization that defines a namespace for now. But it's not necessary that we will need to do that. In general keeping the CLI concise and focused is probably a good thing. In any case, it's fine if we apply explicit namespacing when we need to, rather than doing it proactively by adding overhead in terms of commands and subcommands where it's not required. A bit long explanation to give you the background a bit. I hope this helps.